src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/TimerManager.cs

```diff
@@ -235,7 +251,8 @@ private void ScheduleTimer()
                 }
                 else
                 {
-                    this.UpdateSettingsAndCreateCheckpoint();                    
+                    // Do not update the lastAttemptedTime.
+                    this.UpdateSettingsAndCreateCheckpoint(false);                    
```

Comment:
>false [](start = 59, length = 5)

Why are we passing false here?

Reply:
because in this case, update was not attempted and we don't have to update the last attempted update time.

Reply:
Update Attempt was not completed here.

Reply:
Let's discuss this

---
In reply to: [281155941](https://github.com/microsoft/Service-Fabric-POA/pull/14#discussion_r281155941) [](ancestors = 281155941)